code/modules/jobs/job_types/clown.dm

```diff
@@ -56,4 +56,4 @@
 		return
 
 	H.fully_replace_character_name(H.real_name, pick(GLOB.clown_names)) //rename the mob AFTER they're equipped so their ID gets updated properly.
-	H.dna.add_mutation(CLOWNMUT)
+	H.dna.add_mutation(CLOWNMUT)
```

Comment:
spurious whitespace worsening, actually lots of spurious changes in the job_type folder in general